packages/material-ui/src/Rating/Rating.js

```diff
@@ -73,7 +73,8 @@ const useUtilityClasses = (styleProps) => {
       focusVisible && 'focusVisible',
       readOnly && 'readyOnly',
     ],
-    label: ['label', emptyValueFocused && 'labelEmptyValueActive', 'pristine'],
+    label: ['label', 'pristine'],
+    emptyValueFocused: [emptyValueFocused && 'labelEmptyValueActive'],
```

Comment:
```suggestion
    emptyValueFocused: ['labelEmptyValueActive'],
```

No need for condition here as you are adding this classname anyway on only one label below.

Reply:
tests are failing, but I think I know why we thought it would. Fixing the name.

Reply:
Ah good catch!